streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBStoreTest.java

```diff
@@ -451,17 +453,14 @@ public void shouldReturnUUIDsWithStringPrefix() {
         rocksDBStore.flush();
 
         final KeyValueIterator<Bytes, byte[]> keysWithPrefix = rocksDBStore.prefixScan(prefix, stringSerializer);
-        final List<String> valuesWithPrefix = new ArrayList<>();
         int numberOfKeysReturned = 0;
 
         while (keysWithPrefix.hasNext()) {
-            final KeyValue<Bytes, byte[]> next = keysWithPrefix.next();
-            valuesWithPrefix.add(new String(next.value));
+            keysWithPrefix.next();
             numberOfKeysReturned++;
         }
 
-        assertThat(numberOfKeysReturned, is(1));
-        assertThat(valuesWithPrefix.get(0), is("a"));
```

Comment:
I would still try to keep this assertion, something among the lines of:
```java
if (clashOnPrefix) {
    assertThat(valuesWithPrefix.get(0), either(is("a")).or(is("b")));
} else {
    assertThat(valuesWithPrefix.get(0), is("a"));
}
```

Reply:
SG, I will do that upon merging.